### DISCUSSION TITLE
Eclipse Theia v1.57.0

### DISCUSSION THREAD

**Discussion by jfaltermeier** [eclipse-theia/theia#14630] [Release Announcements] [answered]
Hey everyone 👋,

The Eclipse Theia v1.57.0 release is happening today, starting around 1 PM CET. If you have any nearly-complete PRs that need to be included, let us know in this thread before we begin.

We’ll post updates when the release starts and once it’s done. Please avoid merging any PRs during the release until we confirm it's over.

**Comment by jfaltermeier** [comment]
We will wait for https://github.com/eclipse-theia/theia/pull/14628

**Comment by jfaltermeier** [comment]
The release will start now, We’ll post an update once it has completed.

**Comment by jfaltermeier** [comment, marked as answer]
The release has completed, thank you to everyone that participated and contributed!

**Reply by JonasHelming** [reply]
[News & Noteworthy](https://eclipsesource.com/blogs/2024/12/20/eclipse-theia-1-57-release-news-and-noteworthy/)
